documentation/blog/2023-06-29-svelte-dev-overhaul.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 'svelte.dev: A complete overhaul'
+description: 'The new site comes with accessibility fixes, new features and bottom navbar'
+author: Puru Vijay
+authorURL: https://puruvj.dev
+---
+
+When the initial version of Svelte v3 was released four years ago, it included the single-page documentation that folks are familiar with. That documentation structure has stayed the same as Svelte's API surface has been increasing steadily and more details were added. As a result, the single page got larger and larger to the point where it was becoming difficult to find things. The community has been asking for a revamp, and now it's here!
+
+Meet the new svelte.dev — a complete overhaul of the old website.
+
+## Multi-page docs
+
+The table of contents had grown to be quite large and you had to scroll across half a dozen screens to see the whole thing.
+
+We heard you! The docs are all split up into multiple pages now and all pages list their sections in the righthand sidebar.
+
+All modules exposed by Svelte are also listed in the sidebar under the `Runtime` section:
+
+- [svelte](/docs/svelte)
+- [svelte/store](/docs/svelte-store)
+- [svelte/motion](/docs/svelte-motion)
+- [svelte/transition](/docs/svelte-transition)
+- [svelte/animate](/docs/svelte-animate)
+- [svelte/easing](/docs/svelte-easing)
+- [svelte/action](/docs/svelte-action)
+
+> [svelte/compiler](/docs/svelte-compiler) is under **Compiler and API** section
+
+We've also taken extra care to ensure that all the links from the old website will be redirected to the correct new page.
+
+## Search
+
+The lack of search functionality could make finding stuff a nuisance as <kbd>Ctrl+F</kbd> only returns results in order of occurrence and not order of importance. While <kbd>Ctrl+F</kbd> did have its benefits such as working without JS, now that the site has multiple pages, it's not an option anymore.
```

Comment:
```suggestion
The lack of search functionality could make finding stuff a nuisance as <kbd>Ctrl+F</kbd> only returns results in order of occurrence and not order of importance. While <kbd>Ctrl+F</kbd> did have its benefits such not requiring JS, now that the site has multiple pages, it's not an option anymore.
```

Reply:
Not required? also, grammatical error not adding "as" after "such"